tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,301 @@
+            start_date=start_date,
+            end_date=date.today().strftime('%Y-%m-%d')
+        ).create()
+
+        # Step 10: Publish both the content views and re-read the content views
+
+        # Changing the nailgun timeout time for the rest of the steps as
+        # publish/promote of larger repos take more than 5 minutes
+        entity_mixins.TASK_TIMEOUT = 3600
+        for content_view in (cls.rhel_6_cv, cls.rhel_6_partial_cv):
+            content_view.publish()
+        cls.rhel_6_cv = cls.rhel_6_cv.read()
+        cls.rhel_6_partial_cv = cls.rhel_6_partial_cv.read()
+
+        # Step 11: Promote both content views to 'DEV' and 'QE'
+        for content_view in (cls.rhel_6_cv, cls.rhel_6_partial_cv):
+            assert len(content_view.version) == 1
+            for env in (cls.dev_lce, cls.qe_lce):
+                promote(content_view.version[0], env.id)
+
+        # Step 12: Create host collections
+        for hc_name in ('rhel_6_hc', 'rhel_6_partial_hc'):
+            setattr(
+                cls,
+                hc_name,
+                HostCollection(
+                    organization=cls.org, name=hc_name, max_content_hosts=5
+                ).create()
+            )
+
+        # Step 13: Create activation keys for both content views
+        kwargs = {'organization': cls.org, 'environment': cls.qe_lce.id}
+        rhel_6_ak = ActivationKey(
+            name=u'rhel_6_ak',
+            content_view=cls.rhel_6_cv,
+            host_collection=[cls.rhel_6_hc],
+            **kwargs
+        ).create()
+        rhel_6_partial_ak = ActivationKey(
+            name=u'rhel_6_partial_ak',
+            content_view=cls.rhel_6_partial_cv,
+            host_collection=[cls.rhel_6_partial_hc],
+            **kwargs
+        ).create()
+
+        # Step 14: Assign subscriptions to activation keys
+        # Fetch available subscriptions
+        subs = Subscription(organization=cls.org).search()
+        assert len(subs) > 0
+
+        # Add subscriptions to activation key
+        print 'Adding subscriptions to both Activation keys'
+        for sub in subs:
+            if sub.read_json()['product_name'] == u'Employee SKU':
+                for act_key in [rhel_6_ak, rhel_6_partial_ak]:
+                    act_key.add_subscriptions(data={
+                        u'subscription_id': sub.id
+                    })
+
+        # Step 15: Enable product content in activation keys
+        for act_key in [rhel_6_ak, rhel_6_partial_ak]:
+            act_key.content_override(data={'content_override': {
+                u'content_label': PRD_SETS['rhel6_sat6tools']['label'],
+                u'value': u'1'
+            }})
+
+        # Step 16: Create three client machines and register them to satellite
+        cls.vm = [
+            VirtualMachine(distro='rhel67', tag='incupdate')
+            for _ in range(3)
+        ]
+
+        cls.setup_vm(cls.vm[0], rhel_6_ak.name, cls.org.label)
+        for i in range(1, 3):
```

Comment:
Can you comment here that 1 VM gets an activation-key and the other 2 get a different one because it would represent a specific use case?